api/core/model_runtime/model_providers/bedrock/text_embedding/text_embedding.py

```diff
@@ -28,7 +29,7 @@
 logger = logging.getLogger(__name__)
 
 class BedrockTextEmbeddingModel(TextEmbeddingModel):
-
+    _enc = None
```

Comment:
please define this in the `get_num_tokens` where its needed, instead of the class level